deploy/05_transfer_t.ts

```diff
@@ -8,6 +8,15 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { execute, read } = deployments
 
   const VendingMachineKeep = await deployments.get("VendingMachineKeep")
+  const VendingMachineNuCypher = await deployments.get("VendingMachineNuCypher")
+
+  const vendinMachines = [
+    { tokenSymbol: "KEEP", vendingMachineAddress: VendingMachineKeep.address },
+    {
+      tokenSymbol: "NU",
+      vendingMachineAddress: VendingMachineNuCypher.address,
+    },
+  ]
 
   // There will be 10B T minted on the production environment. The 45% of this
```

Comment:
Could you update the comment? The first two sentences are ok, but rest can be removed after you've added vending machine for NU.

Reply:
Please see d1800a2